### PR TITLE
Improve error messaging around VerifyDevicePath and reduce stutter

### PR DESCRIPTION
**Pull request by davidz627**
/kind cleanup
/assign @msau42 

```release-note
NONE
```

### CODE REVIEW

File: pkg/mount-manager/device-utils.go [hunk 1]
```diff
@@ -214,27 +214,29 @@ func (m *deviceUtils) VerifyDevicePath(devicePaths []string, diskName string) (s
...
 			err := udevadmChangeToDrive(devSDX)
 			if err != nil {
-				return fmt.Errorf("failed to fix disk at path %s which has SCSI ID %s: %v", devSDX, scsiSerial, err)
+				return fmt.Errorf("failed to fix disk which has SCSI ID %s: %v", scsiSerial, err)
 			}
 			return nil
 		}
 	}
-	return fmt.Errorf("udevadm --trigger requested to fix disk %s but no such disk was found in %v", diskName, sds)
+	return fmt.Errorf("udevadm --trigger requested to fix disk %s but no such disk was found in %v", diskName, devToSCSI)
```
Comment (msau42) [inline pkg/mount-manager/device-utils.go]: Are errors like this going to bubble up to the user in k8s events? Do we actually want to give information to users about disks that may not be theirs?

**Comment by davidz627** [comment]
Yah makes sense :)

**Pull request merged**